test/powershell/Modules/Microsoft.PowerShell.Utility/Test-Json.Tests.ps1

```diff
@@ -65,7 +65,7 @@ Describe "Test-Json" -Tags "CI" {
                 errorNode
             }
 "@
-}
+    }
```

Comment:
Incorrect indent earlier in the file was confusing my editor, I can remove this change if policy is to minimize diff